libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -753,30 +753,28 @@ void AP_TECS::_update_pitch(void)
     // A SKE_weighting of 0 provides 100% priority to height control. This is used when no airspeed measurement is available
     // A SKE_weighting of 2 provides 100% priority to speed control. This is used when an underspeed condition is detected. In this instance, if airspeed
     // rises above the demanded value, the pitch angle will be increased by the TECS controller.
-    float SKE_weighting = constrain_float(_spdWeight, 0.0f, 2.0f);
+    float _SKE_weighting = constrain_float(_spdWeight, 0.0f, 2.0f);
```

Comment:
You want to remove the `float` part right? Otherwise you aren't saving the value in the class var.